lib/minify.js

```diff
@@ -36,47 +36,47 @@ export async function minify(name, userOptions) {
     
     check(name);
     
-    const ext = path.extname(name).slice(1);
-    const is = EXT.includes(ext);
+    const type = process.argv.find(arg => EXT.includes(arg.slice(2)))?.slice(2) ?? path.extname(name).slice(1);
```

Comment:
Better place to parse arguments would be [`/bin/minify.js`](https://github.com/coderaiser/minify/blob/e5b37f90390415eb3af6b16f52d41e879cc3643b/bin/minify.js#L53-L54).

What do you think about using autodetecting https://github.com/coderaiser/minify/issues/70 instead?

Reply:
Not sure about autodetection, but I'll look into moving this to bin/minify

Reply:
With help of:

```js
import {fileTypeFromFile} from 'file-type';

console.log(await fileTypeFromFile('Unicorn.png'));
//=> {ext: 'png', mime: 'image/png'}
```

There will be no need to use any flags, filetype will be detected automatically ;)